sass/directives/_headings.scss

```diff
@@ -0,0 +1,57 @@
+// TODO incorporate site-wide typograpy updates
+
+@mixin heading--hero {
+  h1 {
+    font-size: 2.75rem;
+    font-weight: $font-weight-light;
+    margin-bottom: 0;
+  }
+
+  p {
+    margin-top: 28px; // TODO new type spacing (but this may be home specific)
+    font-size: 1.375rem;
+  }
+}
+
+@mixin heading--hero--xl {
+  // In progress - For use with Extra Large header style (https://cb-content-enablement.atlassian.net/wiki/pages/viewpage.action?pageId=43897344)
+  h1 {
+    font-size: 1.375rem;
+    margin-bottom: 0;
+  }
+
+  p {
+    margin-top: 28px;
+    font-size: 2.75rem;
+    font-weight: $font-weight-light;
+  }
+}
+
+@mixin heading--hero--sm {
```

Comment:
Considering that the current convention in place uses 'heading--section', going to go with 'heading--section--sm' to keep everything consistent.